showTasks.cpp

```diff
@@ -1,36 +1,36 @@
-#include <fstream>
-#include <iostream>
-#include <string>
-#include "showTasks.h"
-
-
-void ShowTasks::execute() {
-	system("cls");
-	std::cout << "Showing task(s)" << std::endl;
-
-	std::string taskFileName = "tasks.txt";
-	std::string taskFileLine;
-	std::ifstream taskFile;
-	taskFile.open(taskFileName);
-
-	int taskNumber = 1;
-	if (taskFile.is_open()) {
-		while (taskFile.peek() != EOF) {
-			std::getline(taskFile, taskFileLine);
-			std::cout << taskNumber << "\t" << taskFileLine << std::endl;
-			taskNumber++;
-		}
-		if (taskNumber == 1) {
-			std::cout << "There is no tasks\n\n";
-		}
-	}
-	else {
-		std::cout << "ERROR::showTask::execute::FILE_NOT_OPEN\n";
-	}
-	taskFile.close();
-	std::cout << std::endl;
-}
-
-PossibleState ShowTasks::leave() {
-	return PossibleState::mainMenu;
+#include <fstream>
+#include <iostream>
+#include <string>
+
+#include"config.h"
+#include"showTasks.h"
+
+void ShowTasks::execute() {
+	Config config;
+	system("cls");
+	std::cout << "Showing task(s)" << std::endl;
+	
+	std::string taskFileLine;
+	std::ifstream taskFile;
+	taskFile.open(config.tasksFileName);
+
+	int taskNumber = 1;
```

Comment:
To na pewno trzeba zapisać lepiej.
Bo potem są takie kwiatki jak w linijce 23

Reply:
proponuję wektor
pętla i pushowanie tasków do wektorów

!wektor.empty() -> wypisywanie tasków
else nie ma tasków